From 5fedf142f964f2cb578047b43ea9c6cef77c619c Mon Sep 17 00:00:00 2001
From: "Uwe L. Korn" <uwe.korn@quantco.com>
Date: Mon, 25 Sep 2023 14:41:31 +0200
Subject: [PATCH 1/2] MINGW64 is also a Windows system

---
 configure | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure b/configure
index 4cdcc3c..4008301 100755
--- a/configure
+++ b/configure
@@ -1358,7 +1358,7 @@ echo "${ECHO_T}warning: requested TEA version \"3.7\", have \"${TEA_VERSION}\""
 echo "${ECHO_T}ok (TEA ${TEA_VERSION})" >&6
     fi
     case "`uname -s`" in
-	*win32*|*WIN32*|*CYGWIN_NT*|*CYGWIN_9*|*CYGWIN_ME*|*MINGW32_*)
+	*win32*|*WIN32*|*CYGWIN_NT*|*CYGWIN_9*|*CYGWIN_ME*|*MINGW32_*|*MINGW64_*)
 	    # Extract the first word of "cygpath", so it can be a program name with args.
 set dummy cygpath; ac_word=$2
 echo "$as_me:$LINENO: checking for $ac_word" >&5
-- 
2.39.3 (Apple Git-145)

