From df6b7a198a73af08c17ee015b166e8e9b4c086c1 Mon Sep 17 00:00:00 2001
From: "Uwe L. Korn" <uwe.korn@quantco.com>
Date: Tue, 26 Sep 2023 09:11:59 +0200
Subject: [PATCH 3/3] Remove hardcoded libs

---
 win/makefile.vc | 2 --
 1 file changed, 2 deletions(-)

diff --git a/win/makefile.vc b/win/makefile.vc
index 55f3244..3e0e138 100644
--- a/win/makefile.vc
+++ b/win/makefile.vc
@@ -104,8 +104,6 @@ WISH		= $(DEST_DIR)\bin\wish$(TCL_VER)
 WIN_LIBS	= gdi32.lib user32.lib
 LIBS		= $(DEST_DIR)\lib\tclstub$(TCL_VER).lib \
 		  $(DEST_DIR)\lib\tkstub$(TCL_VER).lib $(WIN_LIBS)
-LIBS		= C:\build\tcl84-64\tclstub$(TCL_VER).lib \
-		  C:\build\tk84-64\tkstub$(TCL_VER).lib $(WIN_LIBS)
 INCLUDES	= -I"$(DEST_DIR)\include" -I. $(include32)
 
 DEFINES = -DDLL_BUILD -DBUILD_Tktable $(TBL_CFLAGS) \
-- 
2.39.3 (Apple Git-145)

